src/commands/sbom/__tests__/fixtures/sbom.1.5.ok.json

```diff
@@ -117,7 +117,7 @@
       ]
     },
     {
-      "bom-ref": "pkg:npm/%40graphql-typed-document-node/core@3.1.1",
+      "bom-ref": "@graphql-typed-document-node/core@3.1.1",
```

Comment:
Please do put change this file. If you want to add another test, please add another file.

Reply:
there are no specific tests for this use case of language detection, the test just ensures the deps were parsed and counts them

creating a new file with just this patch and copying the exact same test seems redundant and excessive - but lmkwyt

Reply:
Thanks for your comment. This file (sbom.1.5.json) is what our internal tooling returns (returns from [osv-scanner](https://github.com/DataDog/osv-scanner)). We should not change it since it's used by [our tests](https://github.com/DataDog/datadog-ci/blob/master/src/commands/sbom/__tests__/payload.test.ts) and test our internal tools. If you want to test another logic, please
1. Add another file
2. Test this file to check this file.

Thank you,